### PR TITLE
Alias release

### PR DESCRIPTION
**Pull request by karravi-msft**
## Description
Changes for Alias Records support in Azure DNS. For swagger - refer Azure/azure-rest-api-specs#3777
---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ x] Please add REST spec PR link to the SDK PR
- [ x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x ] Title of the pull request is clear and informative.
- [ x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
We cannot accept this PR due to the merge commits dating 6 months back.
Please have a clean commit history and we will be able to review/merge this PR.

File: src/SDKs/Dns/Dns.Tests/Dns.Tests.csproj [hunk 1]
```diff
@@ -9,11 +9,15 @@
...
   </PropertyGroup>-->
+  <ItemGroup>
+    <Compile Remove="Properties\AssemblyInfo.cs" />
+  </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.Network" Version="14.0.0-preview" />
     <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.6.0-preview" />
   </ItemGroup>
   <ItemGroup>
+    <!--<ProjectReference Include="..\..\..\SdkCommon\TestFramework\Microsoft.Azure.Test.HttpRecorder\Microsoft.Azure.Test.HttpRecorder.csproj" />-->
```
Comment (dsgouda) [inline src/SDKs/Dns/Dns.Tests/Dns.Tests.csproj]: Why is this commented?

File: src/SDKs/Dns/Dns.Tests/Dns.Tests.csproj [hunk 2]
```diff
@@ -9,11 +9,15 @@
   <!--<PropertyGroup>
     <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
   </PropertyGroup>-->
+  <ItemGroup>
+    <Compile Remove="Properties\AssemblyInfo.cs" />
```
Comment (dsgouda) [inline src/SDKs/Dns/Dns.Tests/Dns.Tests.csproj]: Please remove this line

File: src/SDKs/Dns/Dns.Tests/Dns.Tests.csproj [hunk 3]
```diff
@@ -9,11 +9,15 @@
   <!--<PropertyGroup>
```
Comment (dsgouda) [inline src/SDKs/Dns/Dns.Tests/Dns.Tests.csproj]: Please remove these lines

File: src/SDKs/_metadata/dns_resource-manager.txt [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
 1) azure-rest-api-specs repository information
 GitHub user: Azure
 Branch:      master
-Commit:      58409bca13e129632c26c9d3767f22f922af62dc
+Commit:      5ceb33d2e5c0692dbdbcc907e025c1dfd994f29c
 
 2) AutoRest information
 Requested version: latest
-Bootstrapper version:    C:\Users\muwaqar\AppData\Roaming\npm `-- autorest@2.0.4245  
+Bootstrapper version:    C:\Users\karravi\AppData\Roaming\npm `-- autorest@2.0.4283
```
Comment (dsgouda) [inline src/SDKs/_metadata/dns_resource-manager.txt]: Please pull latest changes from upstream psSdkJson6 and run msbuild build.proj again

**Review by dsgouda — CHANGES_REQUESTED** [review]
Left a few comments

**Comment by shahabhijeet** [comment]
Closing due to https://github.com/Azure/azure-sdk-for-net/pull/4751

**Pull request closed without merging**